distribution/docs/src/main/resources/content/_configuring/connecting-to-sources.adoc

```diff
@@ -0,0 +1,10 @@
+:title: Connecting to Sources
+:type: configuration
+:status: draft
+:parent: Configuring Federation
+:order: 01
```

Comment:
❓ Will this file and `connecting-to-registry.adoc` have different orders when they get published?

Reply:
Yes.